osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/InstallationService.java

```diff
@@ -135,16 +138,50 @@ public void handleCoupleMbusDeviceResponse(final DeviceMessageMetadata deviceMes
         this.handleResponse("coupleMbusDevice", deviceMessageMetadata, result, exception);
     }
 
+    @Transactional(value = "transactionManager")
```

Comment:
is there a reason you need to name the transactionManager? there should only be one, managed by Spring.

Reply:
By convention